### ISSUE TITLE
Sprint assist suddenly lose a lot of speed, need to relog to fix

### ISSUE THREAD

**Issue by McMercy** [MachineMuse/MachineMusePowersuits#783]
I have sprint assist on max, and it works great most of the time, but sometimes i only walk and run half the speed. I have a lot of power and there is no heat. When I disconnect and reconnect I can sprint at full speed again. Im guessing this is a bug? Or is there something im doing wrong?

**Comment by lehjr** [comment]
needs more information

**Comment by McMercy** [comment]
For example when I travelled to end, i lost the ability to run fast. When i logged out and back in i could run in the end again.

**Comment by lehjr** [comment]
No, I mean version information, logs, anything that I can use to replicate the issue. The information you have given me is too vague to replicate or troubleshoot.

**Comment by lehjr** [comment]
Think about it this way, the mod has been around for years, and through that time, there have been different versions of minecraft, forge, even the mod itself, as well as all the other mods it has coexisted with. And it's very common for someone to look for support of an old version of the mod rather than trying the latest version to see if the issue was fixed. So right now, I have no idea what version of the mod you are using, no idea what version of forge you are using, no idea of what other mods you are using, only that you are having an issue in some version of the mod.

**Comment by McMercy** [comment]
Forge version: 1.10.2-forge1.10.2-12.18.3.2281
Powersuits Version: 1.10.2-1.2.110
I made a video showing the bug, + another bug with the tinkertable
https://youtu.be/dd02Y0WR8Bg
Tell me if you need any other information

**Comment by lehjr** [comment]
I'll have to dig into this a bit more. The tinker table is going to be getting a new model as soon as I can come up with something that fits the mod. The sprint assist module... I'll have to do some digging through the old bug reports but I do remember something about that, I think from 1.6.4. 
I did find this: https://github.com/MachineMuse/MachineMusePowersuits/issues/425
The sprint assist is the same as the walking assist, just different lines in the same module. I'll have to experiment with it after the next alpha release.

**Comment by McMercy** [comment]
I also found out that sometimes when I use teleportation commands such as /home, I lose all the energy in the suit. It goes down to 0. Even if im teleporting around in the same dimension.

**Comment by MachiavelliSeraphim** [comment]
Ive experienced the same loss of sprint assist.  
First noticed on my first trip to the nether.  
Burned massive amounts of power when trying to run, but ended up moving much slower.  
Walking assist caused same effect.

ModularPowersuits-1.10.2-1.2.124
Forge 1.10.2-12.18.3.2422
industrialcraft-2-2.6.254-ex110
and a slew of other mods.

Thought id mention that I experienced the same problem.

**Comment by lehjr** [comment]
this should not be happening, as the code was changed. Unfortunately it may be awhile before I have a chance to address it due to work on a rewrite for 1.12 and extreme weather preparations.

**Comment by MachiavelliSeraphim** [comment]
No problem.  Just wanted to echo thats its not an isolated incident.  Good luck with the weather and tge rewrite.
